config/user_preferences_extra_conf.yml.sample

```diff
@@ -0,0 +1,41 @@
+preferences:
+    # the key you can refer to
+    apollo_url_01:
```

Comment:
Would it make sense to remove the sample file from instead? https://github.com/galaxyproject/galaxy/pull/3383/files#diff-5e5bbef6f4c68752f8c1e486e8266d94R57